packages/tally/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Patch Changes
 
+- Optimism chain support
+
+## 1.0.0-alpha.27
+
+### Patch Changes
+
 - Updated dependencies [[`d410b7c`](https://github.com/rabbitholegg/questdk-plugins/commit/d410b7cc98270a9f7c7720f07b07b831176c5221)]:
```

Comment:
This is handled through changesets - [please check this section of the readme](https://github.com/rabbitholegg/questdk-plugins?tab=readme-ov-file#changesets--publishing) for more information

Reply:
fixed, plz, confirm